code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -103,6 +103,9 @@
 			if(affecting.dismember(I.damtype))
 				I.add_mob_blood(src)
 				playsound(get_turf(src), I.get_dismember_sound(), 80, 1)
+		if(I.is_sharp() && prob(probability*1.05) && getorganslot(ORGAN_SLOT_TAIL))

```

Comment:
I multiplied the probability because, wouldn't a tail be slightly less firmly attached than important limbs, like arms/legs?

Reply:
why slightly? Make it 1.5, at least